### PR TITLE
change format to f-string format

### PR DESCRIPTION
**Pull request by mgold1234**
This PR can be merged only after the end of ROSE current cycle.

### CODE REVIEW

**Comment by shiramax** [comment]
@mgold1234 can you please rebase?

File: Pipfile [hunk 1]
```diff
@@ -25,4 +25,4 @@ twisted = "*"
 
 [requires]
 
-python_version = "3.7"
+python_version = "3.6.8"
```
Comment (rollandf) [inline Pipfile:28]: Please revert

**Comment by rollandf** [comment]
@mgold1234 tests are failing

File: rose-admin [hunk 2]
```diff
@@ -39,11 +40,11 @@ class Client(object):
         try:
             method = getattr(self, method_name)
         except AttributeError:
-            self.usage('Unknown verb: %r' % verb)
+            self.usage(f'Unknown verb: {verb}')
         try:
             method()
         except Exception as e:
-            log.error("Error running: %s, %s", verb, e)
+            log.error(f"Error running: {verb}, {e}")
```
Comment (nirs) [inline rose-admin:47]: This is bad change. Logging message should not be formatted. Formatting the messages means we always pay for formatting the logs even when the logs are not enabled (e.g. log.debug() when log level is WARNING).

log.error() is likely always loged, but it does not make sense to handle differently different log messages, so logging calls always pass the arguments to the logger and the logger format the messages before they are written.

Please remove all the changes in logging calls.

**Review by nirs — CHANGES_REQUESTED** [review]

**Pull request closed without merging**